CONTRIBUTING.md

```diff
@@ -0,0 +1,149 @@
+# Contributing guidelines
+
+We thank you in advance :thumbsup: :tada: for taking the time to
+contribute, whether with *code* or with *ideas*, to the Local EGA
+project.
+
+
+## Did you find a bug?
+
+* Ensure that the bug was not already reported by [searching under Issues].
+
+* If you're unable to find an (open) issue addressing the problem, [open a new
+  one]. Be sure to prefix the issue title with **[BUG]** and to include:
+
+  - a *clear* description,
+  - as much relevant information as possible, and
+  - a *code sample* or an (executable) *test case* demonstrating the expected behaviour that is not occurring.
+
+* If possible, use the following [template to report a bug].
+
+
+## Highlevel overview of how we work
+
+### Github projects
+
+We use the [Local EGA](https://github.com/orgs/EGA-archive/projects/3) github
+project in the EGA-Archive to manage our work.
+
+### How we use issues
+
+Issues are the central area for all work.
+
+### Standups
+
+Every day at 10:30 we have standups in Zoom meeting: <id>
+
+### Sprint Reviews and Planning
+
+Approximately every 2 weeks we have sprint reviews and then also a sprint
+planning meeting.
+
+
+
+## How to work on a new feauture/bug
+
+Create an issue on Github, and talk to the team members on the NBIS local-ega
+Slack channel. You can alternatively pick one already created.
+
+Assign yourself to that issue.
+
+Discussions on how to proceed about that issue take place in the comment
+section on that issue, beforehand.
+
+The keyword here is *beforehand*. It is usually a good idea to talk about it
+first. Somebody might have already some pieces in place, we avoid unnecessary
+work duplication and a waste of time and effort.
+
+
+## How we work with Git
+
+All work take place in feature branches. Give your branch a short descriptive
+name and prefix the name with:
+
+   * `feature/` if it is a code feature
```

Comment:
Do we need a `test/` prefix to work on tests? Or it goes to the `feature/`-prefixed branches?

Reply:
I think extending test suite is `feature/`. But now that you mention this, we have no prefix for just plain bugfix (as in, not urgent). Maybe we should have a `bugfix/` prefix, what do other people think?

Reply:
The angular guidelines have quite a few [types](https://github.com/angular/angular/blob/22b96b9/CONTRIBUTING.md#type), and while I don't think we need this many I'd like a set like **docs**, **feature**, **bugfix**, **test**, and maybe **refactor**. I might be overly pedantic, but to me it's descriptive and is easier since there are fewer edge cases with the categories. :)

Reply:
Well, there are several ways of skinning a cat. I don't think I like the **refactor** one, since a refactor should be part of a feature in general (though it should be a separate set of commits). I think it's enough with **bugfix** and **feature** for now, where everything that is not a **bugfix** is a **feature**. We can't really have **hotfixes** yet as there's no production system running.

Reply:
I don't think we need such a complicated branching scheme. We can make use of additional labels to distinguish what type of `/feature` it is.

Reply:
I personally support Martin's idea, I like this branching sheme: **docs**, **feature**, **bugfix**, **test**, and **refactor**.